src/cfi.rs

```diff
@@ -104,7 +103,7 @@ impl<'input, Endian> FallibleIterator for CfiEntriesIter<'input, Endian>
     type Item = CieOrFde<'input, Endian>;
     type Error = Error;
 
-    fn next(&mut self) -> Result<Option<Self::Item>, Self::Error> {
+    fn next(&mut self) -> ::std::result::Result<Option<Self::Item>, Self::Error> {
```

Comment:
We could change these to `Result<Option<Self::Item>>`. Not sure if that's clearer though.